RentACar/RentACar/Helpers/CrudHelper.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+
+namespace RentACar.Helpers
+{
+    static public class CrudHelper
+    {
+        public static List<Car> GetListFromFile(string json)
```

Comment:
minor: parameter `json` is a JSON string or a file path? be careful with names!

Reply:
It's a file path! You are right! Thanks!

Reply:
Your suggestions below about method names (in fact about that, I totally agree) it made me think of the name of this method.. What do you suggest? 😛 maybe _GetListFromFileOrEmptyListIfNotExist_? too long? 😳

Reply:
maybe it's too long, you can clarify such things with a method description, try placing the cursor in the line above the method and typing 3 slashes `///`

Reply:
Nice tip! thanks! i'll change it and if you can in the next PR take a look at the method name again :P

Reply:
we consider a PR "big" enough when more than 500 lines changed.. I would suggest opening a new one if the size is really big or if the approach drastically changed

Reply:
Ohh great! Good advice! Thanks!